canonical-web-and-design/charmhub.io
# Make bundles visible on homepage

## Done
- _Make bundles visible on homepage._
- _Update styling for bundle icons on cards._

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://localhost:8045/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that bundles are visible on homepage

## Issue / Card
Fixes #

## Screenshots
[if relevant, include a screenshot]
